digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressService.scala

```diff
@@ -79,7 +80,15 @@ class RoadAddressService(roadLinkService: RoadLinkService) {
     val viiteRoadLinks = roadLinks.flatMap { rl =>
       val ra = addresses.getOrElse(rl.linkId, Seq())
       buildRoadAddressLink(rl, ra)
+  }
+    val newAnomalousRL = viiteRoadLinks.filter(rl => (rl.administrativeClass == State || rl.roadNumber > 0) && rl.id == 0)
+
+    val missedRL  = withDynTransaction {
+    RoadAddressDAO.getMissingRoadAddresses()
     }
+
+    val missingRL = newAnomalousRL.filterNot(rl => missedRL.exists(mrl => mrl._1 == rl.linkId && mrl._2 == rl.startAddressM && mrl._3 == rl.endAddressM))
```

Comment:
On this case we are just looking at the road address that doesn't exist on the database, so the anomaly will be always code = 1. Shouldn't we just set the code to one on this case?